src/makefiles/cuda_64bit.mk

```diff
@@ -13,5 +13,10 @@ CUDA_FLAGS = --machine 64 -DHAVE_CUDA \
              -std=c++11 -DCUDA_API_PER_THREAD_DEFAULT_STREAM  -lineinfo \
              --verbose -Xcompiler "$(CXXFLAGS)"
 
+ifeq ($(shell test -e $(CUDATKDIR)/lib64/libcudart_static.a && echo -n yes),yes)
```

Comment:
As the conda env install libs, bins, headers of all packages into its root path, the dirname for libs is "lib".

I think we need conda env to prepare a proper toolchain (CXX, MKL, CUDA, etc.) As conda can provide a standalone virtual env like a container of docker.